onebusaway-android/src/main/java/org/onebusaway/android/tripservice/TripService.java

```diff
@@ -137,7 +138,7 @@ public void cancelNotification(int id) {
 
         @Override
```

Comment:
Will do as soon as I am back at a PC.
On May 11, 2016 3:27 PM, "Sean Barbeau" <notifications@github.com> wrote:

> In
> onebusaway-android/src/main/java/org/onebusaway/android/tripservice/TripService.java
> <https://github.com/OneBusAway/onebusaway-android/pull/479#discussion_r62938030>
> :
>
> > @@ -137,7 +138,7 @@ public void cancelNotification(int id) {
> >
> >          @Override
>
> @charles-b-stb <https://github.com/charles-b-stb> Could you please
> completely remove getNotification() here? You'll need to remove it from
> the interface as well.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/OneBusAway/onebusaway-android/pull/479/files/0046a7b34364a5cf55f13dc9bef88949eb012968#r62938030>
>